### DISCUSSION TITLE
Supabse RPC Functions Utils

### DISCUSSION THREAD

**Discussion by williamlmao** [supabase/supabase#19849] [Feature Requests]
I've found it pretty difficult to manage supabase RPC functions, because postgres does not let you update the return type of functions, and the editor only shows the function, and not the entire query including the return type. 

Would be really nice:
- See full query definition of rpc functions, similar to tables in dashboard
- When running db diffs, have a `/rpc` directory similar to edge functions

**Comment by tonypee** [comment]
+1 for an RPC folder, where each file can contain the postgres function. The function could be dropped, and all RPC contac'ed and executed during the migration. That means that any changes will diff correctly. 
I've been thinking of making this myself and simply executing the 'migration' as a post migration task in CI
